persistence/src/main/java/de/cyface/persistence/PersistenceLayer.java

```diff
@@ -745,7 +745,13 @@ private Track collectNextSubTrack(@NonNull final Cursor geoLocationCursor, @NonN
         while (location.getTimestamp() <= pauseEventTime) {
```

Comment:
I would rather add a track.add(location) in front of the while loop and put the !geoLocationCursor.moveToNext() into the loop condition. However I wonder whether the initial location may already be null? Is this possible?